src/main/java/de/ellpeck/rockbottom/api/tile/TilePlant.java

```diff
@@ -0,0 +1,111 @@
+package de.ellpeck.rockbottom.api.tile;
+
+import de.ellpeck.rockbottom.api.StaticTileProps;
+import de.ellpeck.rockbottom.api.entity.Entity;
+import de.ellpeck.rockbottom.api.entity.player.AbstractEntityPlayer;
+import de.ellpeck.rockbottom.api.item.ItemInstance;
+import de.ellpeck.rockbottom.api.tile.state.TileState;
+import de.ellpeck.rockbottom.api.util.BoundBox;
+import de.ellpeck.rockbottom.api.util.Util;
+import de.ellpeck.rockbottom.api.util.reg.ResourceName;
+import de.ellpeck.rockbottom.api.world.IWorld;
+import de.ellpeck.rockbottom.api.world.layer.TileLayer;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Used for two-high plants such as Corn and Cotton
+ */
+public class TilePlant extends TileBasic {
```

Comment:
Rename to TileTallPlant or something or other coz we may want to have 1 tall plants in the future.